src/sass/_banner.scss

```diff
@@ -24,13 +24,21 @@ body > header {
         fill: $medium-monogram-white;
       }
     }
-  }
 
-  .grav-c-nav-menu__external-link {
-    .grav-c-block-link {
-      border-color: transparent;
+    .grav-c-nav-menu, .grav-c-nav-menu__external-link {
+      .grav-c-block-link, .grav-c-block-link:visited {
+        background-color: transparent;
+        border-width: 0;
+        border-color: transparent;
+      }
+
+      .grav-c-block-link:not([href]) {
+        color: #544848;
```

Comment:
This should be using `$grav-co-neutral-asphalt`

Reply:
I completely missed that 😞